deployment/k8s/charts/aggregates/migrationAssistantWithArgo/values.yaml

```diff
@@ -0,0 +1,420 @@
+conditionalPackageInstalls:
+  # Migration infrastructure components
+  etcd: false                    # used by our Argo workflows for workflow coordination. May be used by strimzi clusters too
+  cert-manager: true             # needed by the otel operator
+  kube-prometheus-stack: true    # will be needed for auto-scaling
+  argo-workflows: true           # orchestrates the steps of a migration
+  strimzi-kafka-operator: true   # Required by capture and replay
+  fluent-bit: true
+
+  # Simpler alternative to the opentelemetry-operator.
+  # ONLY set up the otel-collector and require manual instrumentation
+  # Mutually exclusive with otel-collector
+  otel-collector-daemonset: true
+  # EXPERIMENTAL!  Needed for auto-instrumenting and general observability.
+  # Mutually exclusive with otel-collector-daemonset
+  opentelemetry-operator: false
+
+  # Support packages - installed as direct dependencies to this chart as opposed to via the installer job
+  migration-console: true
+
+  # Packages for testing and local development
+  localstack: true
+  # Nice to haves that aren't fully supported yet
+  gatekeeper: false
+  grafana: false
+  jaeger: true
+
+
+snapshotBucketConfiguration:
+  create: true
+  bucketName: snapshot
+  region: us-east-2
+  useLocalStack: true
+  endpoint: localstack
+
+logs:
+  sharedLogsVolume:
+    enabled: true
+    name: logs-pvc
+    size: 10Gi
+    storageClass: ""
+  format: application # application or docker-json
+
+
+installer:
+  image:
+    repository: migrations/migration_console
+    tag: latest
+    pullPolicy: IfNotPresent
+  serviceAccount:
+    create: true
+    name: "helm-installer"
+  rbac:
+    create: true
+
+extraOtelConfiguration:
+  version: "0.124.0"
+  configs:
+    metrics:
+      - prometheus
+      - cloudwatch
+    traces:
+      - jaeger
+      - xray
+
+
+charts:
```

Comment:
I'm assuming these charts should be considered standard charts that we always deploy, whereas an argo workflow will deploy charts like Replayer/RFS. I assume things like localstack will be disabled by default or a separate values.yaml will be made for an EKS setup